metricsx/middleware.go

```diff
@@ -301,13 +322,13 @@ func (sw *Service) UnaryInterceptor(ctx context.Context, req interface{}, info *
 	latency := time.Since(start) / time.Millisecond
 
 	if err := sw.c.Enqueue(analytics.Page{
-		UserId: sw.o.ClusterID,
-		Properties: analytics.
-			NewProperties().
-			SetPath(info.FullMethod).
-			Set("status", status.Code(err)).
-			Set("latency", latency),
-		Context: sw.context,
+		InstanceId:   sw.instanceId,
+		DeploymentId: sw.o.DeploymentId,
+		Project:      sw.o.Service,
+
+		UrlPath:        info.FullMethod,
+		RequestCode:    int(status.Code(err)),
+		RequestLatency: latency.Seconds(),
```

Comment:
I agree, buckets would be the best option here. It will make the processing much easier. We could steps of 10 until 100ms, and then steps of 20 until 500ms, and then steps of 50 until 1s, and then steps of 500ms. Or something like that.

For a quick fix, make this milliseconds :)